### PR TITLE
LPS-147461 Update Congrats Fragment to Mobile and Tablet View

### PR DESCRIPTION
**Pull request by samuelleolopes**

### CODE REVIEW

**Comment by samuelleolopes** [comment]
ci:test:sf

**Review by silasbsi — COMMENTED** [review]
LGTM :+1: @kevenleone could you make another review?

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats/index.css [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
 }
 
 .congrats h4 {
 	color: var(--color-neutral-9);
 	font-size: 16px;
-	font-weight: 500;
 	line-height: 24px;
+}
+
+.you-did-it{
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats/index.css:26]:
```suggestion
.you-did-it {
```

**Review by kevenleone — COMMENTED** [review]
There's just a format thing in the CSS file missing.

But I would like to recommend a better name for `you-did-it` CSS class name

We could use a different name? 

Something like: `congrats-footer`

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats/index.css (hunk 1 above)
Comment (kevenleone) [reply modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats/index.css:26]: What you guys think @samuelleolopes , @silasbsi ??

**Comment by Mylena-Rodrigues** [comment]
@kevenleone, I uploaded the requested changes, please, check if is ok

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats/index.css (hunk 1 above)
Comment (silasbsi) [reply modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats/index.css:26]: make sense.

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**